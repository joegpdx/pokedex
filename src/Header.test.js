import React from 'react';
import renderer from 'react-test-renderer';
import Header from './Header.js';

test('renders Header.js correctly', () => {
  const tree = renderer
    .create(<Header />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});