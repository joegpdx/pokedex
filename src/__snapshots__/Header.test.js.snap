// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Header.js correctly 1`] = `
<header>
  <h1>
    Pokemon Cards
  </h1>
</header>
`;
